### DISCUSSION TITLE
CTAS语法支持吗

### DISCUSSION THREAD

**Discussion by ChPi** [apache/doris#6549] [RFC]
create table t1 as select * from t 
报错
`1064 - errCode = 2, detailMessage = Syntax error in line 1:
create table t1 as select * from t
                ^
Encountered: AS
Expected
, Time: 0.005000s `
我看代码也有CreateTableAsSelectStmt，但是sql parer没找到
如果支持具体语法是什么，麻烦举个例子，谢谢

**Comment by ChPi** [comment]
找到了，正在开发中。
没找到怎么删除这个

**Reply by EmmyMiao87** [reply]
删除哪个？

**Reply by ChPi** [reply]
> 删除哪个？

删除这个讨论。发快了，随后就找到答案了

**Reply by EmmyMiao87** [reply]
^0^

**Comment by morningman** [comment]
No need to delete, you can refer the PR to let others know: #6102
